shared/widgets.py

```
@@ -1419,55 +1431,179 @@ def print_files_with_dialog(paths: list, parent=None, app_context=None) -> None:
+                                    alpha=False,
+                                )
+                                samples = bytes(pix.samples)
+                                preview_cache.append(
+                                    QImage(
+                                        samples, pix.width, pix.height,
+                                        pix.stride, QImage.Format.Format_RGB888,
+                                    ).copy()
+                                )
+                        finally:
+                            doc.close()
+                    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)